examples/example-gateway/build/clients/bar/bar.go

```diff
@@ -448,6 +448,12 @@ func (c *barClient) ArgWithHeaders(
 		headers["x-uuid"] = string(*r.UserUUID)
 	}
 
+	if r.ParamsStruct != nil {
```

Comment:
before the fix, this would have been:
```
if r.ParamsStruct != nil {
 

 
▲
before the fix, this would have been generated as:
```
	if r.ParamsStruct != nil {
		if r.ParamsStruct.UserID != nil {
		}
	}
        headers["user-id"] = string(*r.ParamsStruct.UserID)
```